src/mmr/runner.rs

```diff
@@ -51,26 +66,30 @@ impl Runner {
                 continue;
             }
 
-            if (ptr/30_000 * 30_000) as u64 + 30_000 > epoched && ffi::epoch(epoched) {
-                epoched += 30_000;
+            if (ptr / epoch_length * epoch_length) as u64 + epoch_length > epoched {
+                // notify epoch change
+                if &network == "Mainnet" {
+                    ffi::epoch(epoched);
```

Comment:
heco & bsc do not need to generate DAG files

Reply:
Yes, the `Mainnet` represent ethereum mainnet